src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorSpan.cs

```diff
@@ -651,7 +652,7 @@ public static implicit operator ReadOnlyTensorSpan<T>(TensorSpan<T> span) =>
         /// <param name="indexes">The indexes for the slice.</param>
         /// <returns></returns>
         /// <exception cref="IndexOutOfRangeException">
-        /// Thrown when any index is less than 0 or any index is greater than or equal to FlattenedLength
+        /// Any index is less than 0 or greater than or equal to FlattenedLength.
```

Comment:
I don't know the DocID for this:

```suggestion
        /// Any index is less than 0 or greater than or equal to <c>FlattenedLength</c>.
```